src/bin/vip-import-media.js

```diff
@@ -98,18 +98,29 @@ Are you sure you want to import the contents of the url?
 		'Export any file errors encountered to a JSON file instead of a plain text file',
 		false
 	)
+	.option(
+		'saveErrorLog',
+		'If errors were encountered, save the error log file without prompting',
```

Comment:
Let's do the same here too:

```suggestion
		'Download file-error logs without prompting',
```